build.sbt

```diff
@@ -37,6 +37,8 @@ import scala.build._, VersionUtil._
 // Non-Scala dependencies:
 val junitDep          = "junit"                          % "junit"                            % "4.13.2"
```

Comment:
should this be removed with this upgrade?

Reply:
I just tried to remove junit4 from test:junit module firstly, but now I find we should also remove it from other modules, I should take some more time to solve this.